checks/remotesettings/clients_error_rate.py

```diff
@@ -0,0 +1,125 @@
+"""
+The percentage of reported errors in Uptake Telemetry should be under the specified
+maximum.
+
+For each collection whose error rate is above the maximum, the total number of events
+for each status is returned. The min/max timestamps give the datetime range of the
+dataset obtained from https://sql.telemetry.mozilla.org/queries/64808/
+"""
+import os
+from collections import defaultdict
+from typing import Dict, List
+
+import aiohttp
+
+from poucave.typings import CheckResult
+
+
+EXPOSED_PARAMETERS = ["max_percentage", "min_total_events"]
+
+REDASH_URI = (
+    f"https://sql.telemetry.mozilla.org/api/queries/64808/results.json?api_key="
+)
+REQUESTS_TIMEOUT_SECONDS = int(os.getenv("REQUESTS_TIMEOUT_SECONDS", 5))
+
+
+def sort_dict_desc(d, key):
+    return dict(sorted(d.items(), key=key, reverse=True))
```

Comment:
Can we use `OrderedDict` here?

Reply:
dict are sorted in 3.6+, so I don't think using OrderedDict it's necessary